source/documentation/deploying_services/opensearch.md

```diff
@@ -0,0 +1,281 @@
+
+```sh
+cf update-service SERVICE_NAME -p NEW_PLAN_NAME
+```
+
+where `SERVICE_NAME` is a unique descriptive name for this service instance, and `NEW_PLAN_NAME` is the name of your new plan. For example:
+
+```sh
+cf update-service my-opensearch-service -p medium-ha-1
+```
+
+The plan upgrade will start immediately and finish within an hour. You can check the status of the upgrade by running `cf services`.
+
+<h2 id="remove-the-service">Remove the service</h2>
+
+### Unbind an OpenSearch service from your app
+
+You must unbind the OpenSearch service before you can delete it. Run the following in the command line:
+
+```
+cf unbind-service APP_NAME SERVICE_NAME
+```
+
+where `APP_NAME` is your app's deployed instance name as specified in your app's `manifest.yml` or push command, and `SERVICE_NAME` is a unique descriptive name for this service instance, for example:
+
+```
+cf unbind-service my-app my-es-service
+```
+
+If you unbind your services from your app but do not delete them, the services will persist even after you have deleted your app, and you can re-bind or re-connect to them in future.
+
+### Delete an OpenSearch service
+
+Once you have unbound the OpenSearch service from your app, you can delete the service. Run the following in the command line:
+
+```
+cf delete-service SERVICE_NAME
+```
+
+where `SERVICE_NAME` is a unique descriptive name for this service instance. For example:
+
+```
+cf delete-service my-opensearch-service
+```
+
+Enter `yes` when asked for confirmation.
+
+<h2 id="maintaining-the-service">Maintaining the service</h2>
+
+### Data classification
+
+You can store data classified up to Official on the GOV.UK PaaS. Refer to the [data security classification documentation](/deploying_services/#data-security-classification) for more information.
+
+### OpenSearch backups
+
+Aiven automatically backs up all data stored within any OpenSearch service you create.
+
+Backups are taken every 2 hours. Data is retained for:
+
+- 2 days if you have a `tiny` plan
+- 14 days if you have a `small`, `medium` or `large` plan
+
+To restore data to an earlier state, you can visit the [GOV.UK PaaS support page](https://admin.london.cloud.service.gov.uk/support) or contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+
+### View your OpenSearch data using Kibana
+
+Aiven provide a Kibana user interface through which you can explore the data in your OpenSearch. You can access Kibana using the following instructions:
+
+1. Run the following to get some valid credentials for your OpenSearch:
+
+    ```sh
+    cf create-service-key NAME_OF_YOUR_OPENSEARCH creds-for-kibana
+
+    cf service-key NAME_OF_YOUR_OPENSEARCH creds-for-kibana
+    ```
+
+    That outputs a hostname, username and password that you will use later.
+
+2. Set up an SSH tunnel to your target OpenSearch:
```

Comment:
We should consider making this better using `cf conduit`

Reply:
`cf conduit` gets a mention further down

> This sets up a tunnel so that you can get access, similar to how we use [Conduit](https://github.com/alphagov/paas-cf-conduit) elsewhere.

I assumed there was a difference on how the SSH tunnel is created, but are they equivalent?

Reply:
Standard response here of "this is a copypasta from ES docs, are we updating it individually here and leaving the ES docs bad and dangly?"